src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
I am not saying we should not have default values at all. I am saying the user should be able to customize this value.

Similar logic done [here](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/client.coffee#L228)

 and if not passed in by the user when instantiating the class, we use the defaults to instantiate the class.